src/utils/deploy.ts

```diff
@@ -92,15 +96,38 @@ export class DeployUtils {
 
     deploymentResponse.isSuccess
       ? PostHogUtil.trackEvent('DEPLOY_SUCCESS', {
```

Comment:
suggestion for these tags `DEPLOY_SUCCESS` and so forth to be in constants outside of the logic in the future